helpdesk-bot/src/test/java/org/symphonyoss/symphony/bots/helpdesk/bot/it/HelpDeskBotStories.java

```diff
@@ -102,6 +102,8 @@ public void bootstrap() {
   }
 
   private void prepareEnvironment() {
+    testContext.setCertsDir(System.getProperty("java.io.tmpdir"));
```

Comment:
Add a constant on the class to the location of the certificates.
Private static String CERTS_DIR = "/certs";

and Change the location of certificates to System.getProperty("java.io.tmpdir") + CERTS_DIR